examples/clerk-nextjs/.env.example

```diff
@@ -0,0 +1,5 @@
+# Refer to the README on how to fill these variables
+
+NEXT_PUBLIC_CLERK_FRONTEND_API=
+CLERK_API_KEY=
```

Comment:
Is this needed?

Reply:
Yeah. The Clerk provider needs it, and reads it using the same ENV name as explained in their docs.